<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_AdbProcess</class>
 <widget class="QWidget" name="Dialog_AdbProcess">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>428</width>
    <height>415</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>安卓</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>文件：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_picpath_PC">
       <property name="minimumSize">
        <size>
         <width>160</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>10</width>
         <height>25</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="changeLogo">
       <property name="minimumSize">
        <size>
         <width>115</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>115</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>选择替换的开机动画</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushLogo">
       <property name="minimumSize">
        <size>
         <width>115</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>115</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>修改开机动画</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>设备号：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_head">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>25</height>
        </size>
       </property>
       <property name="maxLength">
        <number>3</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_devid">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>10</width>
         <height>25</height>
        </size>
       </property>
       <property name="maxLength">
        <number>16</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_devid">
       <property name="text">
        <string>自增</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ChangDEVID">
       <property name="minimumSize">
        <size>
         <width>115</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>115</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>修改设备号</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_devid_2">
       <property name="text">
        <string>同时烧录设备号</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="minimumSize">
        <size>
         <width>115</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>115</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>烧录系统</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="restartADB">
       <property name="minimumSize">
        <size>
         <width>115</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>115</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>重启ADB服务</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="iPLabel">
         <property name="text">
          <string>IP:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="iPLineEdit">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>10</width>
           <height>25</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="cOMLabel">
         <property name="text">
          <string>PORT:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="cOMLineEdit">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>10</width>
           <height>25</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_3">
       <property name="minimumSize">
        <size>
         <width>115</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>115</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>主动更新用户列表</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="iDLabel">
         <property name="text">
          <string>用户代码（隐示）：</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="comboBox">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>10</width>
           <height>25</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="unitLabel">
         <property name="text">
          <string>用户单位：</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="unitLineEdit">
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="Label">
         <property name="text">
          <string>用户代码（显示）：</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="LineEdit_IDshow">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>10</width>
           <height>25</height>
          </size>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="Label_2">
         <property name="text">
          <string>用户描述：</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="ComboBox_depict">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>25</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="minimumSize">
        <size>
         <width>115</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>115</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>烧录</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
